Azure/sonic-buildimage
# FG_ECMP support in minigraph.py

- Why I did it
Updated parsing logic in minigraph.py to incorporate FG_ECMP_HASHING feature
- How I did it
Obtain port-ip mapping from IPNextHop section (if it exists in minigraph.xml) --> and ultimately convert it to a ip-end_device mapping to determine which next hop ip corresponds to which bank
- How to verify it
Use an ECMP scenario minigraph.xml to observe new ECMP entries in config_db file. The code is obviously backwards compatible for minigraph.xml w/o ECMP entries

- Description for the changelog

- A picture of a cute animal (not mandatory but encouraged)
